src/web/app/components/modified-timestamps-modal/modified-timestamps-modal.component.html

```diff
@@ -0,0 +1,52 @@
+<p *ngIf="coursesOfModifiedSession">
+  The feedback session has been copied to all courses. However, changes are made to some session timings as the
+  existing timings are no longer valid in these courses: {{ this.coursesOfModifiedSession.join(', ') }}.
+  The changes are as follows:
+</p>
+<p *ngIf="!coursesOfModifiedSession">
```

Comment:
For the same reasons as above, we can check for length here as well:
```
<p *ngIf="coursesOfModifiedSession.length === 0">
```

Reply:
resolved